sunpy/map/maputils.py

```diff
@@ -124,6 +124,17 @@ def sample_at_coords(smap, coordinates):
     return smap.data[smap.wcs.world_to_array_index(coordinates)]
 
 
+def _edge_coordinates(smap):
+    # We need to strip the units from edges before handing it to np.concatenate,
+    # as the .unit attribute is not propagated in np.concatenate for numpy<1.17
+    # When sunpy depends on numpy>=1.17 this unit replacing code can be removed
+    edges = [edge.to_value(u.pix) for edge in map_edges(smap)]
+    # Calculate all the edge pixels
+    edge_pixels = u.Quantity(np.concatenate(edges), unit=u.pix, copy=False)
```

Comment:
You can drop the list comprehension:

```suggestion
    # Calculate all the edge pixels
    edge_pixels = u.Quantity(np.concatenate(edges).value, unit=u.pix, copy=False)
```
works with both old and new numpy's and avoids all copies.

Reply:
I believe that `.value` creates a copy

Reply:
Why would it do that and not just give you the underlying buffer :sob:

Reply:
Don't know if any of this is helpful info: https://docs.astropy.org/en/stable/units/quantity.html#creating-and-converting-quantities-without-copies

Reply:
> Why would it do that and not just give you the underlying buffer 😭

So people don't muck up the underlying buffer?